basic-mathematical-operations.js

```diff
@@ -0,0 +1,16 @@
+function basicOp(operation, value1, value2) {
+  switch (operation) {
+    case "+":
+      return value1 + value2;
+      break;
```

Comment:
If you are returning the result, there is no need to put break. 
the code after  return would be unreachable and will not be used ever